### PR TITLE
refactor: common output_result.rs

### PR DESCRIPTION
**Pull request by kobayu858**
## Description

The output of dag and dagset results are summarized in `util.rs`.
Commonality because it requires the same results

## Related links

None.

## Pre-review checklist for the PR author

- [x] I've confirmed the [self-checklist](https://docs.google.com/spreadsheets/d/1p5KMcm752iCySHJz0xL1qf6fcLDKprbo/edit#gid=863618628).
- [x] The PR is ready for review.

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

### CODE REVIEW

File: lib/src/util.rs [hunk 1]
```diff
@@ -81,6 +85,48 @@ pub fn create_yaml(folder_path: &str, file_name: &str) -> String {
     file_path
 }
 
+#[derive(Serialize, Deserialize)]
```
Comment (atsushi421) [inline lib/src/util.rs:88]: Lines 88~128 should be moved to log.rs.

File: lib/src/util.rs [hunk 2]
```diff
@@ -81,6 +85,48 @@ pub fn create_yaml(folder_path: &str, file_name: &str) -> String {
...
+        period_factor,
+        result,
+    };
+    dump_struct(file_path, &result_info);
+}
+
+#[derive(Serialize, Deserialize)]
+struct DAGSetSchedulerResultInfo {
+    schedule_length: i32,
+    hyper_period: i32,
```
Comment (atsushi421) [inline lib/src/util.rs:112]: These are not important for multi-DAG scheduling and should be removed. :pray: 

```suggestion
```

**Review by atsushi421 — CHANGES_REQUESTED** [review]

File: lib/src/util.rs (hunk 2 above)
Comment (kobayu858) [reply lib/src/util.rs]: Deleted.

File: lib/src/util.rs (hunk 1 above)
Comment (kobayu858) [reply lib/src/util.rs]: Moved.

**Review by atsushi421 — APPROVED** [review]
LGTM

**Pull request merged**